### DISCUSSION TITLE
kbdx format compatibility: missing entries

### DISCUSSION THREAD

**Discussion by shbitsus** [keepassxreboot/keepassxc#6852] [Q&A] [answered]
Hoping for a better UI, I've installed both KeepassX and KeepassXC. In both programs, I'm missing entries that were saved in Keepass2.  Not broken. My mistake.

**Comment by droidmonkey** [comment, marked as answer]
Not sure what to do for you. Can you elaborate further? What makes those entries special?

**Reply by shbitsus** [reply]
Not broken. My mistake. ... and long since forgot what my issue was.
